app/models/good_job/process.rb

```diff
@@ -50,8 +50,10 @@ def self.current_state
         proctitle: $PROGRAM_NAME,
         preserve_job_records: GoodJob.preserve_job_records,
         retry_on_unhandled_error: GoodJob.retry_on_unhandled_error,
-        schedulers: GoodJob::Scheduler.instances.map(&:name),
+        schedulers: GoodJob::Scheduler.instances.map(&:stats),
```

Comment:
@bensheldon What do you think about simply including stats? Might be problematic for backwards compatibility, but not sure how it is currently being used

Reply:
👍🏻